irohad/multi_sig_transactions/gossip_propagation_strategy_prams.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_GOSSIP_PROPAGATION_STRATEGY_PARAMS_HPP
+#define IROHA_GOSSIP_PROPAGATION_STRATEGY_PARAMS_HPP
+
+#include <chrono>
+
+#include <boost/optional.hpp>
+
+// TODO: IR-1317 @l4l (02/05/18) magics should be replaced with options via
+// cli parameters
+static const std::chrono::milliseconds kDefaultPeriod =
```

Comment:
It seems [there's a `constexpr` ctor](https://en.cppreference.com/w/cpp/chrono/duration/duration) for duration, so you may use it instead